### PR TITLE
Feat home page

### PR DESCRIPTION
**Pull request by krishanu2911**
Created the home of the maple coffee 

https://user-images.githubusercontent.com/75560307/154134877-1ce9616b-9296-4c0c-81be-3b745ccde591.mp4

### CODE REVIEW

File: CSS/home.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
+main{
+    width: 100%;
+    height: 100vh;
+    overflow-y: scroll;
+    grid-auto-flow: column;
+    grid-template-rows: 4.5rem auto auto auto auto 8rem;
```
Comment (coldpigli) [inline CSS/home.css]: Suggestion: could use repeat function here

**Review by coldpigli — APPROVED** [review]
Left a small suggestion:

Another suggestion is to use more semantic tags such as <sections> and <articles> instead of <div> everywhere.

File: CSS/home.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/home.css]: Updated with repeat function

File: CSS/home.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: var(--ecom-bg-light);
+}
+.small-nav{
+    display: flex;
+    padding: 1.5rem 0.5rem;
+    justify-content: center;
+    gap: 2rem;
+    align-items: center;
+    background-color: var(--ecom-bg-light);
+    box-shadow: 0 3px 6px #00000029;
```
Comment (tanishq20) [inline CSS/home.css]: You can use var() here for color.

File: CSS/home.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    display: flex;
+    background-color: var(--ecom-bg-light);
+    justify-content: space-around;
+    align-items:flex-start;
+}
+.hero-img{
+    width: 25rem;
+}
+.footer-card > .icons:hover{
+    color: black;
```
Comment (tanishq20) [inline CSS/home.css]: I think you missed the variable color.

**Review by tanishq20 — COMMENTED** [review]
Suggested small change.

File: CSS/home.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: var(--ecom-bg-light);
+}
+.small-nav{
+    display: flex;
+    padding: 1.5rem 0.5rem;
+    justify-content: center;
+    gap: 2rem;
+    align-items: center;
+    background-color: var(--ecom-bg-light);
+    box-shadow: 0 3px 6px #00000029;
```
Comment (shubhamkumar648) [inline CSS/home.css]: suggestion you can also add color as a variable

File: index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+            <a href="#" class=" txt-m semibold-font">Tuesday sale</a>
+            <a href="#" class=" txt-m semibold-font">Limited Edition</a>
+            <a href="#" class=" txt-m semibold-font">Upcomming offers</a>
+        </nav>
+        <div class="banner">
+            <div class="banner-text">
+                <h1 class=" semibold-font ">Experience the smoothness of the most premium <span class="boldest-font">Maple</span> coffee</h1>
+                <a href="#" class="btn txt-lg">ORDER NOW</a>
+            </div>
+            <img class="hero-img" src="./Assets/sample-coffee-2.png" alt="" srcset="">
```
Comment (shubhamkumar648) [inline index.html]: you forgit to write alt attribute vale

File: index.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        </div>
+        <div class="coffee-section">
+            <div class="often-ordered ">
+                <h1>Most Ordered</h1>
+                <div class="coffee-display">
+                    <div class="card card-shadow">
+                        <div class="header">
+                            <div class="header-image-text-div">
+                                    <img class="header-image" src="./Assets/coffee-pro.jpg" alt="" srcset="">
+                            </div>
```
Comment (shubhamkumar648) [inline index.html:71]: same here , alt and srcset

**Review by shubhamkumar648 — APPROVED** [review]
design and good look clean to me! little suggestion I mention in comment, remaining looks good

File: CSS/home.css (hunk 3 above)
Comment (krishanu2911) [reply CSS/home.css]: added the variable thank for pointing just forget to do that

File: CSS/home.css (hunk 4 above)
Comment (krishanu2911) [reply CSS/home.css]: added the variable thank for pointing just forget to do that

File: CSS/home.css (hunk 2 above)
Comment (krishanu2911) [reply CSS/home.css]: Done

File: index.html (hunk 6 above)
Comment (krishanu2911) [reply index.html:71]: I have already updated it in the previous commit just refresh you will be able to see the changes

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**